migrations/migrations/20210518150927-raiting-structures.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const { positions } = require('../../config/constants');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    const { ENUM: enumSequelize } = Sequelize;
+    return Promise.all([
+      queryInterface.addColumn('users', 'position', {
```

Comment:
Migrations should be as atomic as posible, it's better to have two separate migrations (files) if they execute on different entities.